app/assets/stylesheets/semantic-ui/elements/_list.scss

```diff
@@ -135,6 +135,7 @@ ol.ui.list ol li,
 .ui.list .list > .item > .content,
 .ui.list > .item > .content {
   line-height: 1.14285714em;
+  color: $text-color;
```

Comment:
Do you really need to specify a text color here? It doesn't seem to do much in a normal case but will cause problems if you use a list on a light background.